macros/sql/weekdays_between_dates.sql

```diff
@@ -0,0 +1,31 @@
+--code source: https://discourse.looker.com/t/how-to-count-only-weekdays-between-two-dates/3345
+{%- macro weekdays_between_dates(start, stop) -%}
+(
```

Comment:
I like that you thought to wrap the whole thing in parentheses since there's no telling in what context it might be inserted.  However, I think it'd be easier to read/understand that if everything inside the outer parentheses was indented.

Reply:
Done